redirect compose v2 users to github.com/docker/compose (#2065)
docker/compose-cli: README.md

# Docker Compose "Cloud Integrations"

[![Actions Status](https://github.com/docker/compose-cli/workflows/Continuous%20integration/badge.svg)](https://github.com/docker/compose-cli/actions)
[![Actions Status](https://github.com/docker/compose-cli/workflows/Windows%20CI/badge.svg)](https://github.com/docker/compose-cli/actions)

This Compose CLI tool makes it easy to run Docker containers and Docker Compose applications in the cloud using either :
- Amazon Elastic Container Service
([ECS](https://aws.amazon.com/ecs))
- Microsoft Azure Container Instances
([ACI](https://azure.microsoft.com/services/container-instances))
- Kubernetes (Work in progress)

...using the Docker commands you already know.
  

> ## :warning: Compose v2 (a.k.a "Local Docker Compose")

Reviewer: ```suggestion
## :warning: Compose v2 (a.k.a "Local Docker Compose") has Moved
```

> This repository is about "Cloud Integrations", if you're looking for Docker Compose v2,

Reviewer: ```suggestion
This repository is about "Cloud Integrations", the Docker Compose v2
```
